docs/design/coreclr/botr/clr-abi.md

```diff
@@ -816,7 +816,7 @@ Therefore it will expand all indirect calls via the validation helper and a manu
 ## CFG details for x64
 
 On x64, `CORINFO_HELP_VALIDATE_INDIRECT_CALL` takes the call address in `rcx`.
-In addition to the usual registers it also preserves all float registers and `rcx` and `r10`; furthermore, shadow stack space is not required to be allocated.
+In addition to the usual registers it also preserves all float registers and `r10`; furthermore, shadow stack space is not required to be allocated.
```

Comment:
Unintentional change? `rcx` is definitely preserved by the validator (used to actually do the call afterwards).